docs/deployment_patterns/how_to_instantiate_a_data_context_hosted_environments.md

```diff
@@ -4,11 +4,11 @@ title: Deploying Great Expectations in a hosted environment without file system
 
 If you follow the steps of the [Getting Started](../tutorials/getting_started/intro) tutorial, you create a standard deployment of Great Expectations. By default, this relies on two components:
```

Comment:
Just want to confirm that this link and the others are correct - when I try them out on the netlify build, it fails, but they appears to be fine on our currently published docs site.

Reply:
Weird, the Netlify link on the main PR page works for me, and when I was coding I used the Docusaurus yarn server and it worked as well.